skynet.go

```diff
@@ -108,7 +119,29 @@ func UploadFile(path string, opts UploadOptions) (string, error) {
 	return fmt.Sprintf("sia://%s", apiResponse.Skylink), nil
 }
 
-func UploadDirectory(path string, opts UploadOptions) (string, error) {
+func UploadFile(path string, opts FileUploadOptions) (string, error) {
```

Comment:
Same thing here, the type signature of this function has changed, we need to keep it steady for compatibility reasons.